CHANGELOG.md

```diff
@@ -1,10 +1,17 @@
+## 6.20.0 (December 19, 2022).
+
+IMPROVEMENTS:
+
+* Added new datasource: datasource/datasource_artifactory_group.go
+  PR: [#PR](https://github.com/jfrog/terraform-provider-artifactory/pull/607)
```

Comment:
Should include the original issue (https://github.com/jfrog/terraform-provider-artifactory/issues/548)